stream/common/pom.xml

```diff
@@ -39,29 +39,12 @@
     <dependency>
       <groupId>io.grpc</groupId>
       <artifactId>grpc-all</artifactId>
-      <exclusions>
-        <exclusion>
-          <groupId>io.grpc</groupId>
-          <artifactId>grpc-netty-shaded</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>org.bouncycastle</groupId>
-          <artifactId>bcpkix-jdk15on</artifactId>
```

Comment:
@shoothzj It existed here before.

Reply:
it was added in #2631, however I guess it's obsolete now.

Reply:
removed in grpc-java in v1.48: https://github.com/grpc/grpc-java/commit/4d00e0cb. I'll remove it also from this PR.